blueocean-git-pipeline/src/main/java/io/jenkins/blueocean/blueocean_git_pipeline/GitBareRepoReadSaveRequest.java

```diff
@@ -164,6 +165,7 @@ <T> T invokeOnScm(final GitSCMFileSystem.FSFunction<T> function) throws IOExcept
         }
     }
 
+    @SuppressFBWarnings(value = "DCN_NULLPOINTER_EXCEPTION", justification = "Empty repository throws NPE")
```

Comment:
```suggestion
    @SuppressFBWarnings(value = "DCN_NULLPOINTER_EXCEPTION", justification = "TODO check if this NPE is still real, and if so fix it")
```

Reply:
resolved in e4f44aba58fa7baf959f7a5b30aad54f599c566e